### ISSUE TITLE
Javascript causes fatal error evaluating this.parent._automaticallyAdjustsScrollViewInsets on iOS

### ISSUE THREAD

**Issue by tsonevn** [NativeScript/nativescript-angular#1460]
_From @Alessandro994 on July 26, 2018 10:29_

> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
Opening a model from a ListView item makes the app crash.

### Which platform(s) does your issue occur on?
iOS 11.4 (probably on iOS > 10)

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** 3.5.10
- **CLI:** 4.1.2
- **Cross-platform modules:** 4.1.1
- **Runtime(s):** tns-ios: version 4.1.1

### Please tell us how to recreate the issue in as much detail as possible.
When clicking on an item of a ListView inside a RadSideDrawer, the app crashes.
I attach below the snippet of code including the content of the drawer. When I tap on the label of the item having the bmi field, I get the following error:

> ***** Fatal JavaScript exception - application has been terminated. *****
> Native stack trace:
> 1   0x10923d649 NativeScript::reportFatalErrorBeforeShutdown(JSC::ExecState*, JSC::Exception*, bool)
> 2   0x109265c5b NativeScript::FFICallback<NativeScript::ObjCMethodCallback>::ffiClosureCallback(ffi_cif*, void*, void**, void*)
> 3   0x109b5b5c1 ffi_closure_unix64_inner
> 4   0x109b5bf22 ffi_closure_unix64
> 5   0x10f52d456 -[CALayer layoutSublayers]
> 6   0x10f531667 CA::Layer::layout_if_needed(CA::Transaction*)
> 7   0x10f4b80fb CA::Context::commit_transaction(CA::Transaction*)
> 8   0x10f4e579c CA::Transaction::commit()
> 9   0x10c362f2c _afterCACommitHandler
> 10  0x10e56b607 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
> 11  0x10e56b55e __CFRunLoopDoObservers
> 12  0x10e54fb81 __CFRunLoopRun
> 13  0x10e54f30b CFRunLoopRunSpecific
> 14  0x111358a73 GSEventRunModal
> 15  0x10c33a057 UIApplicationMain
> 16  0x109b5bd7d ffi_call_unix64
> 17  0x1330c7100
> JavaScript stack trace:
> 1   onMeasure@file:///app/tns_modules/tns-core-modules/ui/scroll-view/scroll-view.js:122:46
> 2   measure@file:///app/tns_modules/tns-core-modules/ui/core/view/view.js:54:27
> 3   measureChild@file:///app/tns_modules/tns-core-modules/ui/core/view/view-common.js:815:26
> 4   layoutView@file:///app/tns_modules/tns-core-modules/ui/core/view/view.js:590:26
> 5   viewDidLayoutSubviews@file:///app/tns_modules/tns-core-modules/ui/core/view/view.js:632:27
> 6   UIApplicationMain@[native code]
> 7   start@file:///app/tns_modules/tns-core-modules/application/application.js:264:26
> 8   run@file:///app/tns_modules/tns-core-modules/application/application.js:284:10
> 9   bootstrapNativeScriptApp@file:///app/tns_modules/nativescript-angular/platform-common.js:157:26
> 10  bootstrapApp@file:///app/tns_modules/nativescript-angular/platform-common.js:91:38
> 11  bootstrapModule@file:///app/tns_modules/nativescript-angular/platform-common.js:80:26
> 12  anonymous@file:///app/main.js:15:57
> 13  evaluate@[native code]
> 14  moduleEvaluation@[native code]
> 15  @[native code]
> 16  promiseReactionJob@[native code]
> JavaScript error:
> file:///app/tns_modules/tns-core-modules/ui/scroll-view/scroll-view.js:122:46: JS ERROR TypeError: undefined is not an object (evaluating 'this.parent._automaticallyAdjustsScrollViewInsets')

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
```
  <StackLayout tkDrawerContent class="stack-bg">

    <Button class="btn btn-rounded-lg btn-primary" (tap)="onToggleScoreList()" text="Chiudi" marginTop="12"></Button>

    <ListView [items]="details" class="list-group list-score">
      <ng-template let-item="item">
        <GridLayout columns="*, auto, 72, 48" rows="*" class="list-group-item bg-primary">
          <Label col="0" row="0" class="list-score-label" [text]="item.label"></Label>
          <Label col="1" row="0" *ngIf="!item.bmi" class="list-score-label"></Label>
          <Label col="1" row="0" *ngIf="item.bmi" class="bmi-label icon-mdi" [text]="iconInfo" (tap)="onOpenBmiDialog(item)"></Label>
          <Label col="2" row="0" class="list-score-value-label m-r-24" [text]="item.points"></Label>
          <Image col="3" row="0" [src]="item?.state ? 'res://score_' + item?.state : 'res://icon'" class="score-state-img" loadMode="async">
          </Image>
        </GridLayout>
      </ng-template>
    </ListView>
  </StackLayout>
```

_Copied from original issue: telerik/nativescript-ui-feedback#750_

**Comment by tsonevn** [comment]
Hi @Alessandro994,
I tested this case in a sample project, however, was unable to recreate the reported problem. I checked the scenario with iOS 9.0. 
![screencast 2018-07-27 at 9 35 32 am](https://user-images.githubusercontent.com/17448734/43305489-a8819be8-9180-11e8-81a3-bb4df357066c.gif)

I am also attaching sample project, please review it and make the needed changes, which will allow us to recreate the issue.
[Archive.zip](https://github.com/telerik/nativescript-ui-feedback/files/2234738/Archive.zip)

**Comment by Alessandro994** [comment]
Apologize for the lack of information!
First of all: probably I was not clear enough when I explained the problem. I got the error when I tried to open a modal (and not model as I incorrectly wrote) by clicking on an item of a ListView inside a RadSideDrawer. And only on iOS versions equal or grater than 10. 
While I was applying the changes to the sample I noticed also that the issue was caused by the template of the aforementioned modal custom component, which was wrapped inside a ScrollView.
I've temporarily removed the ScrollView as I think it is no more required, but next week I'll try anyway to reproduce the issue and let you know further details.
